setup.py

```diff
@@ -84,17 +84,25 @@ def find_javac(possible_homes):
     for home in possible_homes:
         for javac in [join(home, name), join(home, 'bin', name)]:
             if exists(javac):
+                if sys.platform == "win32" and not PY2: # Avoid path space execution error
+                    return'"%s"' % javac
                 return javac
     return name  # Fall back to "hope it's on the path"
 
 
 def compile_native_invocation_handler(*possible_homes):
     '''Find javac and compile NativeInvocationHandler.java.'''
     javac = find_javac(possible_homes)
-    subprocess.check_call([
-        javac, '-target', '1.6', '-source', '1.6',
-        join('jnius', 'src', 'org', 'jnius', 'NativeInvocationHandler.java')
-    ])
+    try:
+        subprocess.check_call([
+            javac, '-target', '1.6', '-source', '1.6',
+            join('jnius', 'src', 'org', 'jnius', 'NativeInvocationHandler.java')
+        ])
+    except FileNotFoundError:
+        subprocess.check_call([
+            javac.replace('""', ''), '-target', '1.6', '-source', '1.6',
```

Comment:
wouldn't it be better to do this replacement all the time? there is no sense in keeping an empty quote in the initial path, no? and it would look better than try/except.

Reply:
Sorry this replace suppose to be **.replace('"', '')** instead the version commented, the try-except is for try perform subprocess method on the old way on some system does not require add **"** (e.g when use powershell)

Reply:
I understand your point, let's try to do on a better way